statsmodels/tsa/holtwinters.py

```diff
@@ -944,12 +1250,12 @@ def _predict(self, h=None, smoothing_level=None, smoothing_slope=None,
         level = lvls[1:nobs + 1].copy()
         if use_boxcox or use_boxcox == 'log' or isinstance(use_boxcox, float):
             fitted = inv_boxcox(fitted, lamda)
-            level = inv_boxcox(level, lamda)
```

Comment:
I don't really understand why `level`, `slope` and `season` are "back-transformed" here. For the simulations I need the values in the transformed space, therefore I commented them for now.

The tests are still running fine, does this mean we can remove this?

Reply:
That is interesting. I can't see why the back-transformation would be sensible, and I notice that R's forecast package does not back-transform the states.

The only question here is about backwards compatibility - if people are relying on the current behavior. But it looks like the current behavior might be best classified as a bug?

Reply:
Yes, I would also classifiy this as a bug. 
What would be the best way to handle this? I could, for a start, store the original values in a private attribute and use these for the simulations. Then we could open a separate issue and PR to discuss backwards compability issues.

Reply:
What you suggest is definitely the ideal way forward, but a little annoying for you. If you don't mind doing that, that would be great.